VK/VK/GroupNewsScreen/GroupTableViewController.swift

```diff
@@ -0,0 +1,32 @@
+// GroupTableViewController.swift
```

Comment:
```suggestion
// GroupNewsTableViewController.swift
```
мне как - то исправили и такое)))